### ISSUE TITLE
More details for Include and then Include

### ISSUE THREAD

**Issue by voroninp** [dotnet/EntityFramework.Docs#2521]
It does not follow from the documentation what the behavior of `Include` and `ThenInclude` regarding `null`s of collection navigation properties. If included set of values is empty, will navigation property stay `null` or will it be initialized with empty collection?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bbd36f35-e257-d0ed-8f28-f539ddcb71ec
* Version Independent ID: 093903d5-f7c7-5a78-c0e5-0e2662dcedef
* Content: [Loading Related Data - EF Core](https://docs.microsoft.com/en-us/ef/core/querying/related-data)
* Content Source: [entity-framework/core/querying/related-data.md](https://github.com/dotnet/EntityFramework.Docs/blob/master/entity-framework/core/querying/related-data.md)
* Product: **entity-framework**
* Technology: **entity-framework-core**
* GitHub Login: @rowanmiller
* Microsoft Alias: **avickers**

**Comment by ajcvickers** [comment]
@voroninp Whenever Include is used on a collection the collection will be created but empty if there are no related items.

**Comment by voroninp** [comment]
@ajcvickers Yep, but it is worth describing this behavior, though I'd expect it by default anyway.
